changelogs/unreleased/gh-3682-uri-encode-decode.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/uri
```

Comment:
Regarding the documentation request:

> `uri.unescape()` accepts a string that will be decoded and optionally a
> table with decoding options: string with unreserved symbols that will
> *not* be decoded and boolean option that enables/disables decoding of
> '+' as a space character. Table with default decoding options is defined
> as `uri.RFC3986`.

`uri.unescape()` unescapes all `%HH` occurences disregarding of `opts.unreserved`. The function only looks at `opts.plus`. It is the correct behavior. The documentation request should be adjusted accordingly to don't confuse anyone.